apps/astarte_realm_management/test/astarte_realm_management/queries_test.exs

```diff
@@ -419,79 +391,70 @@ defmodule Astarte.RealmManagement.QueriesTest do
 
     endpoint =
       find_endpoint(
-        client,
         "com.ispirata.Hemera.DeviceLog.Status",
         2,
         "/filterRules/%{ruleId}/%{filterKey}/value"
       )
 
-    endpoint_id = endpoint[:endpoint_id]
-
+    endpoint_id = endpoint.endpoint_id
     interface_id = Astarte.Core.CQLUtils.interface_id("com.ispirata.Hemera.DeviceLog.Status", 2)
 
-    assert endpoint[:interface_name] == "com.ispirata.Hemera.DeviceLog.Status"
-    assert endpoint[:interface_major_version] == 2
-    assert endpoint[:interface_minor_version] == 1
-    assert endpoint[:interface_type] == 1
-    assert endpoint[:allow_unset] == true
-    assert endpoint[:value_type] == 7
+    assert endpoint.interface_name == "com.ispirata.Hemera.DeviceLog.Status"
+    assert endpoint.interface_major_version == 2
+    assert endpoint.interface_minor_version == 1
+    assert endpoint.interface_type == :properties
+    assert endpoint.allow_unset == true
+    assert endpoint.value_type == :string
 
-    query =
-      DatabaseQuery.new()
-      |> DatabaseQuery.statement(@insert_devicelog_status_0)
-      |> DatabaseQuery.put(:interface_id, interface_id)
-      |> DatabaseQuery.put(:endpoint_id, endpoint_id)
+    params = %{interface_id: interface_id, endpoint_id: endpoint_id}
+    Repo.query!(@insert_devicelog_status_0, params)
+    Repo.query!(@insert_devicelog_status_1, params)
 
-    DatabaseQuery.call!(client, query)
+    device_id_test = "536be249-aaaa-4e02-9583-5a4833cbfe49"
```

Comment:
This should be a UUID as defined in the `IndividualProperty` table (i.e. an `Astarte.DataAccesss.UUID`)